tests/unit/steps/test_hypervisor.py

```diff
@@ -390,3 +496,119 @@ def test_hypervisor_upgrade_plan_single_machine(model):
     plan = planner.generate_upgrade_plan(target, False)
 
     assert str(plan) == exp_plan
+
+
+def test_hypervisor_upgrade_plan_some_units_upgraded(model):
+    """Testing generating hypervisors upgrade plan partially upgraded."""
+    target = OpenStackRelease("victoria")
+    exp_plan = dedent_plan(
+        """
+    Upgrading all applications deployed on machines with hypervisor.
```

Comment:
Please use `\` at the beginning, so we can remove the part removing new line from `dedent_plan` helper function. More you can see [here](https://docs.python.org/3/library/textwrap.html#textwrap.dedent).